src/lightning/app/cli/commands/cd.py

```diff
@@ -0,0 +1,95 @@
+import os
+from typing import Optional
+
+import click
+from rich.live import Live
+from rich.spinner import Spinner
+from rich.text import Text
+
+from lightning.app.cli.commands.connection import _LIGHTNING_CONNECTION_FOLDER
+from lightning.app.utilities.app_helpers import Logger
+
+logger = Logger(__name__)
+
+_HOME = os.path.expanduser("~")
+_CD_FILE = os.path.join(_LIGHTNING_CONNECTION_FOLDER, "cd.txt")
+
+
+@click.argument("path", required=False)
+def cd(path: Optional[str] = None) -> str:
+    """Command to nagivate through the Lightning Cloud filesystem."""
```

Comment:
Not sure what navigate through means, maybe more explicit:
```suggestion
    """Change the current directory within the Lightning Cloud filesystem."""
```

Reply:
Done 1